cli/internal/helm/charts/edgeless/constellation-services/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: constellation-services
+description: A chart to deploy all microservices that are part of a valid constellation cluster
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
+
+dependencies:
+  - name: kms
+    version: "0.1.0"
```

Comment:
How are you determining these version numbers?
I think they should be tied to our current Constellation release version, e.g. v2.2.0-pre on main, and v2.2 .0 for our next release